### PR TITLE
Remove "Feedback" page

### PR DESCRIPTION
**Pull request by kpyszkowski**
Ref: https://github.com/threshold-network/bitcoin-on-base/issues/1
This pull request removes "Feedback" page and related components.

### CODE REVIEW

File: src/components/FeedbackAnalyticsInfo.tsx [hunk 1]
```diff
@@ -1,70 +0,0 @@
...
-  Stack,
-  useColorModeValue,
-  StackProps,
-} from "@threshold-network/components"
-import sunglassesLight from "../static/images/SunglassesLight.png"
-import sunglassesDark from "../static/images/SunglassesDark.png"
-import diagonalPillLight from "../static/images/DiagonalPillLight.png"
-import diagonalPillDark from "../static/images/DiagonalPillDark.png"
-import diagonalArrowLight from "../static/images/DiagonalArrowLight.png"
-import diagonalArrowDark from "../static/images/DiagonalArrowDark.png"
```
Comment (michalsmiarowski) [inline src/components/FeedbackAnalyticsInfo.tsx:16]: Let's also remove those images from the repo

File: src/enums/modal.ts [hunk 2]
```diff
@@ -31,9 +31,7 @@ export enum ModalType {
   ConfirmDeauthorization = "CONFIRM_DEAUTHORIZATION",
   DeauthorizationCompleted = "DEAUTHORIZATION_COMPLETED",
   DeauthorizationInitiated = "DEAUTHORIZATION_INITIATED",
-  Analytics = "ANALYTICS",
```
Comment (michalsmiarowski) [inline src/enums/modal.ts:34]: Can we additionaly remove `analyicsReport` url from `externalHref.ts`?

https://github.com/threshold-network/bitcoin-on-base/blob/7a6d3149a786db8c6d1ad199dc22fd5c668d55c6/src/enums/externalHref.ts#L24

It's not used anywhere and it looks like it never was 😅

File: src/components/Modal/AnalyticsModal/index.tsx [hunk 3]
```diff
@@ -1,81 +0,0 @@
...
-  Divider,
-} from "@threshold-network/components"
-import { BaseModalProps } from "../../../types"
-import withBaseModal from "../withBaseModal"
-import { useModal } from "../../../hooks/useModal"
-import { ModalType } from "../../../enums"
-import { FeedbackSubmissionType } from "../FeedbackSubmissionModal"
-import FeedbackAnalyticsInfo from "../../FeedbackAnalyticsInfo"
-import analyticsImageLight from "../../../static/images/AnalyticsIllustration.png"
-import analyticsImageDark from "../../../static/images/AnalyticsIllustrationDark.png"
```
Comment (michalsmiarowski) [inline src/components/Modal/AnalyticsModal/index.tsx:23]: Let's remove those two images from the repo.

File: src/pages/Overview/AnalyticsBanner.tsx [hunk 4]
```diff
@@ -1,23 +0,0 @@
-import { FC } from "react"
-import { useColorModeValue } from "@threshold-network/components"
-import analyticsImageLight from "../../static/images/AnalyticsIllustration.png"
-import analyticsImageDark from "../../static/images/AnalyticsIllustrationDark.png"
-import { ModalType } from "../../enums"
-import { useModal } from "../../hooks/useModal"
-import SecondaryAnnouncementBanner from "../../components/AnnouncementBanner/SecondaryAnnouncementBanner"
```
Comment (michalsmiarowski) [inline src/pages/Overview/AnalyticsBanner.tsx:7]: Looks like `SecondaryAnnoucementBanner` is not used anywhere else, so maybe we could remove it too?

Make sure that it's not used though and also remove a theme for that if my assumption is correct.

**Review by michalsmiarowski — CHANGES_REQUESTED** [review]
Grate job overall! 🔥 Left some comments to look at before the merge.

Let's also resolve the conflicts with `main` branch.

File: src/pages/Overview/AnalyticsBanner.tsx (hunk 4 above)
Comment (kpyszkowski) [reply src/pages/Overview/AnalyticsBanner.tsx:7]: Ref commit: 4b1a7238

File: src/components/Modal/AnalyticsModal/index.tsx (hunk 3 above)
Comment (kpyszkowski) [reply src/components/Modal/AnalyticsModal/index.tsx:23]: Ref commit: 080cff06

File: src/enums/modal.ts (hunk 2 above)
Comment (kpyszkowski) [reply src/enums/modal.ts:34]: Ref commit: de5c51c9

File: src/components/FeedbackAnalyticsInfo.tsx (hunk 1 above)
Comment (kpyszkowski) [reply src/components/FeedbackAnalyticsInfo.tsx:16]: Ref commit: 080cff06

**Review by michalsmiarowski — APPROVED** [review]
LGTM

**Pull request merged**